server/src/main/java/org/apache/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -35,10 +38,30 @@ public void run(DataSegment segment, SegmentActionHandler handler)
     handler.replicateSegment(segment, getTieredReplicants());
   }
 
-  protected static void validateTieredReplicants(final Map<String, Integer> tieredReplicants)
+  /**
+   * Function to create a tiered replicants map choosing default values, in case the map is null.
+   * {@code useDefaultTierForNull} decides the default value.
+   * <br>
+   * If the boolean is true, the default value is a singleton map with key {@link DruidServer#DEFAULT_NUM_REPLICANTS}
+   * and value @{@link DruidServer#DEFAULT_TIER}.
+   * <br>
+   * If the boolean is false, the default value is an empty map. This will enable the new behaviour of not loading
+   * segments to a historical unless the tier and number of replicants are explicitly specified.
+   */
+  protected static Map<String, Integer> createTieredReplicants(final Map<String, Integer> tieredReplicants, boolean useDefaultTierForNull)
   {
-    if (tieredReplicants.size() == 0) {
-      throw new IAE("A rule with empty tiered replicants is invalid");
+    if (useDefaultTierForNull) {
+      return Configs.valueOrDefault(tieredReplicants, ImmutableMap.of(DruidServer.DEFAULT_TIER, DruidServer.DEFAULT_NUM_REPLICANTS));
+    } else {
+      return Configs.valueOrDefault(tieredReplicants, ImmutableMap.of());
+    }
+  }
+
+  protected static void validateTieredReplicants(final Map<String, Integer> tieredReplicants, boolean useDefaultTierForNull)
+  {
+    if (tieredReplicants.size() == 0 && useDefaultTierForNull) {
+      // If useDefaultTierForNull is true, null is translated to a default tier, and an empty replicant tier map is not allowed.
+      throw new IAE("A rule with empty tiered replicants is invalid unless \"useDefaultTierForNull\" is set to false.");
     }
     for (Map.Entry<String, Integer> entry : tieredReplicants.entrySet()) {
       if (entry.getValue() == null) {
```

Comment:
Try using this exception in the next line instead:

```suggestion
      throw InvalidInput.exception("Number of replicas in tier [%s] is null", entry.getKey());
```

Reply:
Similarly for the case `entry.getValue() < 0`, use this instead:

```suggestion
      throw InvalidInput.exception("Number of replicas for tier [%s] must be positive. Found [%d].", entry.getKey(), entry.getValue());
```

Reply:
Changed these exception types.